Helper/Data.php

```diff
@@ -246,8 +246,22 @@ public function getModes()
     public function getCaptureModes()
     {
         return [
-            'auto' => 'immediate',
-            'manual' => 'manual'
+            'auto' => 'Immediate',
+            '1_7days' => '1 - 7 days',
+            'manual' => 'Manual'
+        ];
+    }
+
+    /**
+     * return recurring types for configuration setting
+     * @return array
+     */
+    public function getPosCaptureModes()
+    {
+        return [
+            'auto' => 'Immediate',
+            '1min_2days' => '1 min - 2 days',
```

Comment:
Do we have a follow up ticket for this?

Reply:
Since this field is only checked in one place I already adjusted the usage:
https://github.com/Adyen/adyen-magento2/blob/f9efb40c74caa3db1b09bd225876887ed1591a90/Model/Cron.php#L1472